src/perception/PoseEstimatorModule.cpp

```diff
@@ -39,101 +39,99 @@ bool PoseEstimatorModule::detectObjects(
   // Checks detected objects, looks up the database,
   // and adds new skeletons to the world env
   bool any_detected = false;
-  visualization_msgs::MarkerArrayConstPtr marker_message
+  visualization_msgs::MarkerArrayConstPtr markerMessage
       = ros::topic::waitForMessage<visualization_msgs::MarkerArray>(
           mMarkerTopic, mNodeHandle, timeout);
 
   // Making sure the Message from a pose estimator is non empty
-  if (marker_message == nullptr)
+  if (markerMessage == nullptr)
   {
     dtwarn << "[PoseEstimatorModule::detectObjects] nullptr Marker Message "
            << mMarkerTopic << std::endl;
     return false;
   }
 
-  if (marker_message->markers.empty())
+  if (markerMessage->markers.empty())
   {
     dtwarn << "[PoseEstimatorModule::detectObjects] No markers on topic "
            << mMarkerTopic << std::endl;
     return false;
   }
 
-  dart::utils::DartLoader urdf_loader;
+  dart::utils::DartLoader urdfLoader;
 
-  for (const auto& marker_transform : marker_message->markers)
+  for (const auto& markerTransform : markerMessage->markers)
   {
     // TODO: Add DELETE_ALL Functionality
     // TODO: Update when we move over to ROS Kinetic. Indigo doesn't
     // have the enum value.
-    // if (marker_transform.action == visualization_msgs::Marker::DELETEALL) {
-    if (marker_transform.action == 3)
+    // if (markerTransform.action == visualization_msgs::Marker::DELETEALL) {
+    if (markerTransform.action == 3)
     {
       dtwarn << "[PoseEstimatorModule::detectObjects] We cannot currently "
                 "handle DELETE_ALL markers."
              << std::endl;
       continue;
     }
-    const auto& marker_stamp = marker_transform.header.stamp;
-    const auto& obj_id = marker_transform.id;
-    const auto& obj_ns = marker_transform.ns;
-    const auto& detection_frame = marker_transform.header.frame_id;
-    if (!timestamp.isValid() || marker_stamp < timestamp)
+    const auto& markerStamp = markerTransform.header.stamp;
+    const auto& objectId = markerTransform.id;
+    const auto& objectName = markerTransform.ns;
+    const auto& detectionFrame = markerTransform.header.frame_id;
+    if (!timestamp.isValid() || markerStamp < timestamp)
     {
       continue;
     }
 
-    const std::string obj_uid = obj_ns + "_" + std::to_string(obj_id);
-
     // Initialize a DetectedObject class for this object
-    DetectedObject this_object(
-        obj_uid, obj_ns, detection_frame, marker_transform.text);
+    DetectedObject thisObject(
```

Comment:
This needs a try-catch, warning, and continue. Otherwise, a bad marker from a rogue node can shut down the program.